src/utils/logHealthCheck.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * helper function to do log health check
+ */
+const { logInfo, logSystemEventAsync} = require('../operations/common/logging');
+const { AdminLogger } = require('../admin/adminLogger');
+
+module.exports.handleLogHealthCheck = async () => {
+    let healthy = true;
+    try {
+        logInfo('Log HealthCheck', {});
+        const adminLogger = new AdminLogger();
+        await adminLogger.logInfo('Admin Log HealthCheck');
```

Comment:
AdminLogger doesn't use the ES/mongo logs cluster, so we can remove this.

Reply:
removed

Reply:
removed